### DISCUSSION TITLE
Does the library React-native-ranimated-2 require allowed newArchEnabled flag

### DISCUSSION THREAD

**Discussion by YOEL311** [reactwg/react-native-new-architecture#111] [Q&A] [answered]
I use ``react-native-reanimated-2`` 

I understand that this library uses ``TurboModules`` and ``Hermes engine``

> Reanimated 2 is in an early version. As we wanted to share it with the community as soon as we could, the library still has some rough edges and limitations that we plan to address soon. Unfortunately some of the limitations come from the immaturity of React Native's **TurboModule** architecture that Reanimated 2 relies on. As a consequence we won't be able to support older versions of React Native and some issues that we yet plan to resolve may require full support of **TurboModules** which is not yet available to the public.

[source][1]

However, their documentation does not mention that you should enable it.

Further confusing is the fact that the value is false in the example project. 

It is explicitly written in the comment that it should be set to true in order to allow libraries to use it. 

````
# Use this property to enable support to the new architecture.
# This will allow you to use TurboModules and the Fabric render in
# your application. You should enable this flag either if you want
# to write custom TurboModules/Fabric components OR use libraries that
# are providing them.
newArchEnabled=false
````
[Official React-native-reanimated-2 example][2]

I am very confused. 

  [1]: https://docs.swmansion.com/react-native-reanimated/docs/next/#known-problems-and-limitations
  [2]: https://github.com/software-mansion/react-native-reanimated/blob/main/Example/android/gradle.properties

**Comment by frankcalise** [comment, marked as answer]
reanimated 3 is for the new arch, source: https://blog.swmansion.com/announcing-reanimated-3-16167428c5f7
